### PR TITLE
Update CHANGES.

### PR DESCRIPTION
**Pull request by jjnicola**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -7,6 +7,26 @@ or get the entire source code repository and view log history:
...
 $ cd openvas-scanner && git log
 
+openvas-scanner 6.0.0 (unreleased)
+
+Main changes compared to openvas-scanner 6.0+beta2:
+* Function to get the currently running script filename has been added.
+* Debugging nasl mechanism has been improved, replacing preprocessor directives
+  with g_debug facility.
+* An issue related to the log facility and greenbone-nvt-sync has been fixed
+  improved.
```
Comment (wiegandm) [inline CHANGES]: `fixed improved.` does not make much sense, I assume this should be `fixed.`?

File: CHANGES [hunk 2]
```diff
@@ -7,6 +7,26 @@ or get the entire source code repository and view log history:
...
+* OpenVAS reload has been improved.
+* Code related to redis queries was improved.
+* An issued which caused failed nasl-lint in case of unneeded nested functions
```
Comment (wiegandm) [inline CHANGES]:
```suggestion
* An issue which caused nasl-lint to fail in case of unneeded nested functions
```

File: CHANGES [hunk 3]
```diff
@@ -7,6 +7,26 @@ or get the entire source code repository and view log history:
...
+  has been addressed.
+* An issue which caused returning erroneous values by
+  get_plugin_preference() has been addressed.
+* Issues reported by static code analyzer has been addressed.
```
Comment (wiegandm) [inline CHANGES]:
```suggestion
* Issues reported by static code analyzer have been addressed.
```

Comment (mattmundell) [reply CHANGES]:
```suggestion
* Issues reported by static code analysis have been addressed.
```

**Pull request merged**